paperqa/agents/search.py

```diff
@@ -455,6 +455,18 @@ async def maybe_get_manifest(
 FAILED_DOCUMENT_ADD_ID = "ERROR"
 
 
+def get_manifest_kwargs(
+    manifest: dict[str, Any], manifest_fallback_location: str, file_location: str
+) -> dict[str, Any]:
```

Comment:
Few comments:

1. Can you move this to be next to `maybe_get_manifest` in the file?
2. Can you put `file_location` first in the args, since it's primary over "fallback location"
3. Can you rename to something more explicit, like `fetch_kwargs_from_manifest`